pipelines/science.yaml

```diff
@@ -6,6 +6,9 @@ tasks:
     class: pfs.drp.stella.fitPfsFluxReference.FitPfsFluxReferenceTask
     config:
       minBroadbandFluxes: 1
+      minTeff: 0
+      maxTeff: 9999
+      paramMargin: [0, 0, 0, 0]
```

Comment:
Why are the integration test flux standards being rejected? Is there something wrong with them, or is it something wrong with the code?

Reply:
"gen2" test succeeds even if I replace fluxCalibrate.py with pfsFluxReference.py. Effective temperature of the three FLUXSTD are estimated to be ~7000K. But "gen3" test fails with the default configurations because effective temperature of the three FLUXSTD are 5500K, 8000K, and nan K. All the three FLUXSTD are rejected.
Sha1 hashes of `pfsArm` differ between gen2 and gen3. I have not investigated further.